### PR TITLE
Update weave-kube docs for k8s  1.6

### PR DESCRIPTION
**Pull request by bboreham**

### CODE REVIEW

**Comment by awh** [comment]
#### Documentation changes:
modified [kube-addon](https://dev-weavewww.pantheonsite.io/docs/net/weave-kube-1.6-docs/kube-addon/)

File: site/kube-addon.md [hunk 1]
```diff
@@ -51,11 +58,25 @@ Shut down Kubernetes, and _on all nodes_ perform the following:
...
-The URL [https://git.io/weave-kube](https://git.io/weave-kube) points
+The URLs https://git.io/weave-kube and https://git.io/weave-kube-1.6 point
 to the YAML file for the latest release of the Weave Net addon.
 Historic versions are archived on our [GitHub release
 page](https://github.com/weaveworks/weave/releases).
 
+## Upgrading Kubernetes to version 1.6
+
+In version 1.6, Kubernetes has increased security, so we need to
+create a special service account to run Weave Net.  This is done in
```
Comment (brb) [inline site/kube-addon.md]: Nitpick: s/  / /

File: site/kube-addon.md [hunk 2]
```diff
@@ -51,11 +58,25 @@ Shut down Kubernetes, and _on all nodes_ perform the following:
...
+the file `weave-daemonset-k8s-1.6.yaml` attached to the release.
```
Comment (marccarre) [inline site/kube-addon.md]: Suggestion:
```
attached to the [Weave Net release](https://github.com/weaveworks/weave/releases/latest).
```
which is more specific, as we talk about both Kubernetes and Weave Net in this paragraph.

Comment (brb) [inline site/kube-addon.md]: Not related to this PR, but `weave-daemonset-k8s-1.6.yaml` contains not only the DaemonSet definition. Wouldn't it make sense to rename it to e.g. `weave-k8s-1.6.yaml`?

File: site/kube-addon.md [hunk 3]
```diff
@@ -22,6 +22,13 @@ with a single command:
 kubectl apply -f https://git.io/weave-kube
 ```
 
+**Note:** If you are using Kubernetes 1.6 or later, please use this
+  command instead:
+
+```
+kubectl apply -f https://git.io/weave-kube-1.6
+```
```
Comment (marccarre) [inline site/kube-addon.md]: Suggestion:
```
Weave Net can be installed onto your CNI-enabled Kubernetes cluster
with a single command:

* Kubernetes versions up to `1.5.*`:

    kubectl apply -f https://git.io/weave-kube

* Kubernetes versions `1.6` and above:

    kubectl apply -f https://git.io/weave-kube-1.6
```
given it is more a choice for the end user than a note which may seem optional.

Comment (rade) [inline site/kube-addon.md]: I'd swap those.

**Comment by awh** [comment]
#### Documentation changes:
modified [kube-addon](https://dev-weavewww.pantheonsite.io/docs/net/weave-kube-1.6-docs/kube-addon/)

**Pull request merged**